pkg/tenant/members.go

```diff
@@ -35,6 +84,57 @@ func (s *Server) MemberList(c *gin.Context) {
 	c.JSON(http.StatusNotImplemented, "not implemented yet")
 }
 
+// MemberCreate adds a new member to the database and returns
+// a 201 StatusCreated response.
+//
+// Route: /member
+func (s *Server) MemberCreate(c *gin.Context) {
+	var (
+		err    error
+		member *api.Member
+	)
+
+	// Bind the user request and return a 400 response if binding
+	// is not successful.
+	if err = c.BindJSON(&member); err != nil {
+		log.Warn().Err(err).Msg("could not bind member create request")
+		c.JSON(http.StatusBadRequest, api.ErrorResponse("could not bind request"))
+		return
+	}
+
+	// Verify that a member id does not exist and return a 400 response if
+	// the member id exists.
+	if member.ID != "" {
+		c.JSON(http.StatusBadRequest, api.ErrorResponse("member is cannot be specified on create"))
+		return
+	}
+
+	// Verify that a member name exists and return a 400 response if
+	// the member name does not exist.
+	if member.Name == "" {
+		c.JSON(http.StatusBadRequest, api.ErrorResponse("member name is required"))
+		return
+	}
+
+	if member.Role == "" {
+		c.JSON(http.StatusBadRequest, api.ErrorResponse("member role is required"))
+		return
+	}
+
+	m := &db.Member{
+		Name: member.Name,
+		Role: member.Role,
+	}
```

Comment:
The user will have to supply tenant ID here, but we need to parse it as a ULID into the `db.Member` struct.